### PR TITLE
Corrected a few typos in the documentation and comments

### PR DESCRIPTION
**Pull request by tgiphil**
Just fixing up a few typos as I read/browse/study through the source code.

### CODE REVIEW

File: src/jit/codegenlinear.cpp [hunk 1]
```diff
@@ -590,7 +590,7 @@ void CodeGen::genCodeForBBlist()
             case BBJ_THROW:
                 // If we have a throw at the end of a function or funclet, we need to emit another instruction
                 // afterwards to help the OS unwinder determine the correct context during unwind.
-                // We insert an unexecuted breakpoint instruction in several situations
+                // We insert an executed breakpoint instruction in several situations
```
Comment (stephentoub) [inline src/jit/codegenlinear.cpp]: Why did this change?

File: src/jit/codegenxarch.cpp [hunk 2]
```diff
@@ -162,7 +162,7 @@ void CodeGen::genEmitGSCookieCheck(bool pushReg)
         //    cookie check as part of tail call codegen. GenExitCode() needs to special case
         //    fast tail calls implemented as epilog+jmp or such tail calls should always get
         //    dispatched via helper.
-        // 3) Materialize GS cookie check as a sperate node hanging off GT_CALL node in
+        // 3) Materialize GS cookie check as a seperate node hanging off GT_CALL node in
```
Comment (stephentoub) [inline src/jit/codegenxarch.cpp]: It's spelled `separate`

Comment (tgiphil) [reply src/jit/codegenxarch.cpp]: Both fixed.

**Comment by tgiphil** [comment]
Don't merge. I'll rebase with a final set.

**Comment by stephentoub** [comment]
> Don't merge. I'll rebase with a final set.

We generally just "squash and merge".

**Pull request merged**